ang/civicase/activity/activity-forms/services/activity-form-services/email-activity-form.service.js

```diff
@@ -29,7 +29,14 @@
      * @returns {string} the form URL for activities that are sent emails.
      */
     function getEmailActivityFormUrl (activity) {
-      return getActivityFormUrl(activity, { action: 'view' });
```

Comment:
@deb1990 why did we stop using the `getActivityFormUrl` service? This service is used in some other parts of the app and might have the exact same issue because of it. Is it possible to confirm that anyone using `getActivityFormUrl` is not breaking as well? And the fix should probably go to that service.

Reply:
yes @reneolivo I checked, it was only used in 2 places. One here, and in one another place. But the other area is behaving as expected. So no issues.